lightning/src/ln/channelmanager.rs

```diff
@@ -7671,10 +7672,20 @@ where
 						pending_claim: PendingMPPClaimPointer(Arc::clone(pending_claim)),
 					}
 				});
+
+				// Create new attribution data as the final hop. Always report a zero hold time, because reporting a
+				// non-zero value will not make a difference in the penalty that may be applied by the sender.
+				let attribution_data = process_fulfill_attribution_data(
+					None,
+					&htlc.prev_hop.incoming_packet_shared_secret,
+					0,
```

Comment:
Can we check for this behavior in the existing test, i.e. the recipient will always set the final hold time to 0?

Reply:
Added to the `test_fulfill_hold_times`